### DISCUSSION TITLE
Ansible says offending appears appears here ,  even after giving the Correct syntax

### DISCUSSION THREAD

**Discussion by Keerthana-Practice** [learndevopsonline/learndevopsonline#1886] [Q&A]
Hi Raghu ,
When am running the Frontend ansible , am getting the syntax error "offending appears appears here " even after providing the Correct syntax

![image (6)](https://github.com/learndevopsonline/learndevopsonline/assets/164914309/eac76cfb-6a85-4ff1-acc2-48fb5b8a028c)

**Comment by chaithanya1812** [comment]
Hi @Keerthana-Practice 
##Issue:-  check your code common role was not there...
